noir-projects/aztec-nr/aztec/src/note/utils.nr

```diff
@@ -1,26 +1,32 @@
-use crate::{context::PrivateContext, note::{note_header::NoteHeader, note_interface::NoteInterface}};
+use crate::{
+    context::PrivateContext, generators::G_slot,
+    note::{note_header::NoteHeader, note_interface::NoteInterface}
+};
 
 use dep::protocol_types::{
-    constants::GENERATOR_INDEX__INNER_NOTE_HASH,
     hash::{
-    pedersen_hash, compute_unique_note_hash, compute_siloed_note_hash as compute_siloed_note_hash,
+    compute_unique_note_hash, compute_siloed_note_hash as compute_siloed_note_hash,
     compute_siloed_nullifier as compute_siloed_nullifier_from_preimage
 },
-    utils::arr_copy_slice
+    point::Point, utils::arr_copy_slice
 };
+use dep::std::{embedded_curve_ops::multi_scalar_mul, hash::from_field_unsafe};
+
+pub fn compute_slotted_note_hash_from_preimage(storage_slot: Field, note_hiding_point: Point) -> Field {
```

Comment:
I called this function like that since I needed to differentiate it from `compute_slotted_note_hash`. But I agree that it's a confusing name so I renamed it to `compute_slotted_note_hash_raw` in https://github.com/AztecProtocol/aztec-packages/pull/7618/commits/f6038788e2430fa5bd66c31b60329d203af4c323 

Could not think of a better name.